src/js/node/util/Types.hx

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (C)2014-2019 Haxe Foundation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+package js.node.util;
+
+/**
+	`Types` provides a number of type checks for different kinds of built-in objects.
+	Unlike `Syntax.instanceof` or `Object.prototype.toString.call(value)`, these checks do not
+	inspect properties of the object that are accessible from JavaScript (like their prototype),
+	and usually have the overhead of calling into C++.
+
+	The result generally does not make any guarantees about what kinds of properties or behavior
+	a value exposes in JavaScript. They are primarily useful for addon developers who prefer to do
+	type checking in JavaScript.
+**/
+@:jsRequire("util", "types")
+extern class Types {

```

Comment:
Shouldn't all the methods here be declared as `static`?

Reply:
Thnaks for your review comment! I fixed it.